src/java/neroxis/map/SCMap.java

```diff
@@ -376,6 +381,138 @@ public void addCubeMap(CubeMap cubeMap) {
+
+        FloatMask[] texturesMasks = getTextureMasksScaled(symmetrySettings);
+        FloatMask oldLayer1 = texturesMasks[0];
+        FloatMask oldLayer2 = texturesMasks[1];
+        FloatMask oldLayer3 = texturesMasks[2];
+        FloatMask oldLayer4 = texturesMasks[3];
+        FloatMask oldLayer5 = texturesMasks[4];
+        FloatMask oldLayer6 = texturesMasks[5];
+        FloatMask oldLayer7 = texturesMasks[6];
+        FloatMask oldLayer8 = texturesMasks[7];
+
+        setTextureMasksLow(new BufferedImage(newMapBoundsSize, newMapBoundsSize, BufferedImage.TYPE_INT_ARGB));
+        setTextureMasksHigh(new BufferedImage(newMapBoundsSize, newMapBoundsSize, BufferedImage.TYPE_INT_ARGB));
+
+        oldLayer1.min(0f).max(1f).setSize2(resizeCurrentMapContentTo, true);
+        oldLayer2.min(0f).max(1f).setSize2(resizeCurrentMapContentTo, true);
+        oldLayer3.min(0f).max(1f).setSize2(resizeCurrentMapContentTo, true);
+        oldLayer4.min(0f).max(1f).setSize2(resizeCurrentMapContentTo, true);
+        oldLayer5.min(0f).max(1f).setSize2(resizeCurrentMapContentTo, true);
+        oldLayer6.min(0f).max(1f).setSize2(resizeCurrentMapContentTo, true);
+        oldLayer7.min(0f).max(1f).setSize2(resizeCurrentMapContentTo, true);
+        oldLayer8.min(0f).max(1f).setSize2(resizeCurrentMapContentTo, true);
+
+        FloatMask layer1 =  new FloatMask(newMapBoundsSize, new Random().nextLong(), symmetrySettings);
+        layer1.addWithOffset(oldLayer1, locToPutCenterOfCurrentMapContent, true);
+        FloatMask layer2 =  new FloatMask(newMapBoundsSize, new Random().nextLong(), symmetrySettings);
+        layer2.addWithOffset(oldLayer2, locToPutCenterOfCurrentMapContent, true);
+        FloatMask layer3 =  new FloatMask(newMapBoundsSize, new Random().nextLong(), symmetrySettings);
+        layer3.addWithOffset(oldLayer3, locToPutCenterOfCurrentMapContent, true);
+        FloatMask layer4 =  new FloatMask(newMapBoundsSize, new Random().nextLong(), symmetrySettings);
+        layer4.addWithOffset(oldLayer4, locToPutCenterOfCurrentMapContent, true);
+        FloatMask layer5 =  new FloatMask(newMapBoundsSize, new Random().nextLong(), symmetrySettings);
+        layer5.addWithOffset(oldLayer5, locToPutCenterOfCurrentMapContent, true);
+        FloatMask layer6 =  new FloatMask(newMapBoundsSize, new Random().nextLong(), symmetrySettings);
+        layer6.addWithOffset(oldLayer6, locToPutCenterOfCurrentMapContent, true);
+        FloatMask layer7 =  new FloatMask(newMapBoundsSize, new Random().nextLong(), symmetrySettings);
+        layer7.addWithOffset(oldLayer7, locToPutCenterOfCurrentMapContent, true);
+        FloatMask layer8 =  new FloatMask(newMapBoundsSize, new Random().nextLong(), symmetrySettings);
+        layer8.addWithOffset(oldLayer8, locToPutCenterOfCurrentMapContent, true);
+
+        setTextureMasksLowScaled(layer1, layer2, layer3, layer4);
+        setTextureMasksHighScaled(layer5, layer6, layer7, layer8);
```

Comment:
This still seems wrong to me. seems like if the image method doesn't work for textureMasks it wouldn't work for the other images either. Would be better if the issue could be resolved and it all be done with the images.

Reply:
Same with the heightMap I just didn't drag far enough

Reply:
The texture images seem to be stored differently with variable sizes.  I have literally resizing dozens of times now across a large variety of maps, and the resizer works well.  Images like the heightMap appear to work just fine this way.